[2023-08-24 09:20:05,988] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\leticia.gerola\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
